cmake/pcl_options.cmake

```diff
@@ -42,6 +42,10 @@ mark_as_advanced(PCL_NO_PRECOMPILE)
 option(PCL_ENABLE_SSE "Enable or Disable SSE optimizations." ON)
 mark_as_advanced(PCL_ENABLE_SSE)
 
+# Allow the user to disable ccache if ccache is available
+option(PCL_DISABLE_CCACHE "Disable using ccache for compilation" OFF)
```

Comment:
Is there a particular reason to have a negated option name? What about `PCL_ENABLE_CCACHE` defaulting to `ON`? Additionally, I propose to make the creation of the option conditional on the generator type. As you pointed out, VS does not support `ccache`. Thus the option does not make sense.

Reply:
`ENABLE_CCACHE` without ccache doesn't make sense. That's why I chose disable over enable

Reply:
> VS does not support ccache

Installing ccache in Windows is usually via mingw (or similar) and I don't know if CMake can access over this weird boundary. I don't know what happens with Windows, which is partly why I want volunteers 😆